### PR TITLE
Make PrintX10Metrics available

### PR DESCRIPTION
**Pull request by asuhan**

### CODE REVIEW

File: Sources/x10/swift_bindings/XLATensor.swift [hunk 1]
```diff
@@ -1069,3 +1069,7 @@ extension XLATensor {
     }
   }
 }
+
+public func PrintX10Metrics() {
+  x10_xla_tensor_wrapper.PrintMetrics()
```
Comment (compnerd) [inline Sources/x10/swift_bindings/XLATensor.swift:1074]: I think that you can get away with `PrintMetrics()` but I may be mistaken.

**Comment by compnerd** [comment]
Oh, and did you ensure that `PrintMetrics` is properly annotated for export on Windows from the C++ side?

File: Sources/x10/swift_bindings/XLATensor.swift (hunk 1 above)
Comment (asuhan) [reply Sources/x10/swift_bindings/XLATensor.swift]: Done.

**Comment by asuhan** [comment]
@compnerd It has `XLA_API` already, do we need anything else?
@saeta I don't know if we can test this effectively from this repository, the main concern is visibility. The ultimate test would be to call it from swift-models, I think.

**Comment by compnerd** [comment]
`XLA_API` is exactly what is needed, if it has it already, then nothing to do here, move along :)

**Pull request merged**